org-code-javabuilder/theater/src/main/java/org/code/Image.java

```diff
@@ -0,0 +1,67 @@
+package org.code;
```

Comment:
NIT: This will be `org.code.theater`

Reply:
What if I wanted to use it in the Park?

Reply:
We can call it something different than "theater" but the package cannot be org.code. It needs a more descriptive name.

Reply:
Recommendations? This will also be true for `SoundLoader`

Reply:
Perhaps `org.code.image` and `org.code.sound`? The curriculum team may have opinions here.

Reply:
`org.code.media` seems to be acceptable to everyone